### PR TITLE
To xfail pfcwd_warm_reboot on 7050cx3 in 202305

### PR DESCRIPTION
**Pull request by Gfrom2016**
### Description of PR

Summary:
Fixes # (issue)
Add xfail on 7050cx3 in 202305+ images.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### Documentation

Signed-off-by: zitingguo-ms <zitingguo@microsoft.com>

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -782,6 +782,7 @@ pfcwd/test_pfcwd_warm_reboot.py:
      conditions:
         - "'dualtor' in topo_name"
         - https://github.com/sonic-net/sonic-mgmt/issues/8400
+        - "hwsku in ['Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8'] and release in ['202305', 'master']"
```
Comment (ZhaohuiS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:785]: @Gfrom2016 in our nightly test, I found release is  "release": "unknown" in fact.
Could you please check if it should be 202305 or 20230531?

Comment (Gfrom2016) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:785]: This has been fix in 20230531.03, release is '202305'

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Comment by congh-nvidia** [comment]
Hi @Gfrom2016 , why do we need the new condition for the XFAIL? The test is failing on all platforms and all branches, isn't it?
Now with the new condition, the test is enabled on Nvidia platform and failing.

**Comment by Gfrom2016** [comment]
@congh-nvidia this has been fixed in #10075

**Comment by congh-nvidia** [comment]
> @congh-nvidia this has been fixed in #10075

Thanks!

**Pull request merged**